lte/gateway/deploy/roles/agw_docker/tasks/main.yml

```diff
@@ -144,6 +144,14 @@
   tags:
     - agwc
 
+- name: Restart ovs
+  ignore_errors: yes
+  shell: |
+    ifdown gtp_br0
+    /etc/init.d/openvswitch-switch  force-reload-kmod
```

Comment:
There is script to update OVS, lets use that `ovs-kmod-upgrade.sh`

Reply:
fixed